third_party/tarantool_ev.h

```diff
@@ -63,7 +63,7 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
-extern const ev_tstamp TIMEOUT_INFINITY;
+#define TIMEOUT_INFINITY 365*86400*100.0
```

Comment:
The value should be enclosed in parentheses.

Reply:
Done.